docs/dataset_fields_for_iris_timestamp.md

```diff
@@ -0,0 +1,112 @@
+# Dataset Fields for IRIS Timestamp
```

Comment:
I think this is a bit confusing, maybe something more like 'Dataset and Timestamp Reference' 
Maybe rename the file too